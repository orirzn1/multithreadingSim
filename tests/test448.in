L9
S8
O15
N6

T0
I@0x00000000
STORE $6, $2, $7
SUBI $1, $0, 0x4
ADDI $3, $6, 0x3
ADD $7, $0, $0
ADD $5, $2, $2
ADDI $4, $0, 0x4
LOAD $4, $0, $0
STORE $2, $5, $4
HALT $0

T1
I@0x00000000
ADD $2, $2, $0
SUBI $3, $4, 0x4
SUB $1, $7, $2
SUBI $7, $0, 0x3
ADDI $4, $2, 0x1
ADDI $5, $0, 0x2
ADDI $2, $1, 0x1
HALT $0

T2
I@0x00000000
ADDI $5, $6, 0x2
STORE $6, $0, $2
SUB $0, $2, $6
STORE $7, $6, 0x1
SUB $7, $2, $5
ADD $3, $2, $5
HALT $0

T3
I@0x00000000
SUB $6, $2, $1
ADDI $6, $3, 0x2
ADDI $3, $3, 0x1
LOAD $5, $6, $2
STORE $0, $0, $0
HALT $0

T4
I@0x00000000
LOAD $0, $5, 0x0
STORE $7, $0, 0x1
ADD $7, $3, $3
SUBI $1, $5, 0x4
SUBI $6, $7, 0x1
STORE $6, $1, 0x3
HALT $0

T5
I@0x00000000
SUB $0, $7, $4
SUBI $4, $4, 0x2
LOAD $1, $5, 0x2
LOAD $2, $7, $3
LOAD $3, $1, $3
HALT $0

D@0x0
0x9
0x3
0xA
0x8
0x9
0x6
0x1
0x6
0x2
