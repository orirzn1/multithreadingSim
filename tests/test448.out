
---- Blocked MT Simulation ----

Register file thread id 0:
	R0 = 0x0	R1 = 0xFFFFFFFC	R2 = 0x0	R3 = 0x3	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 1:
	R0 = 0x0	R1 = 0x0	R2 = 0x1	R3 = 0xFFFFFFFC	R4 = 0x1	R5 = 0x2	R6 = 0x0	R7 = 0xFFFFFFFD
Register file thread id 2:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x2	R4 = 0x0	R5 = 0x2	R6 = 0x0	R7 = 0xFFFFFFFE
Register file thread id 3:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x1	R4 = 0x0	R5 = 0x0	R6 = 0x2	R7 = 0x0
Register file thread id 4:
	R0 = 0x0	R1 = 0xFFFFFFFC	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0xFFFFFFFF	R7 = 0x0
Register file thread id 5:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0xFFFFFFFC	R4 = 0xFFFFFFFE	R5 = 0x0	R6 = 0x0	R7 = 0x0
Blocked MT CPI for this program 7.279070

-----Finegrained MT Simulation -----

Register file thread id 0:
	R0 = 0x0	R1 = 0xFFFFFFFC	R2 = 0x0	R3 = 0x3	R4 = 0x0	R5 = 0x0	R6 = 0x0	R7 = 0x0
Register file thread id 1:
	R0 = 0x0	R1 = 0x0	R2 = 0x1	R3 = 0xFFFFFFFC	R4 = 0x1	R5 = 0x2	R6 = 0x0	R7 = 0xFFFFFFFD
Register file thread id 2:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x2	R4 = 0x0	R5 = 0x2	R6 = 0x0	R7 = 0xFFFFFFFE
Register file thread id 3:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x1	R4 = 0x0	R5 = 0x0	R6 = 0x2	R7 = 0x0
Register file thread id 4:
	R0 = 0x0	R1 = 0xFFFFFFFC	R2 = 0x0	R3 = 0x0	R4 = 0x0	R5 = 0x0	R6 = 0xFFFFFFFF	R7 = 0x0
Register file thread id 5:
	R0 = 0x0	R1 = 0x0	R2 = 0x0	R3 = 0x0	R4 = 0xFFFFFFFE	R5 = 0x0	R6 = 0x0	R7 = 0x0
Finegrained Multithreading CPI for this program 1.139535

